CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # PLCrashReporter Change Log
 
+## Version 1.8.2 (Under development)
+
+* **[Fix]** Fix `double-quoted` warnings in Xcode 12.
```

Comment:
```suggestion
* **[Fix]** Fix `double-quoted imports` warnings in Xcode 12.
```

Reply:
The full warning looks `Double-quoted include "SameHeader.h" in framework header, expected angle-bracketed instead.`, so I am not sure that use `imports` will be correct in this case. Besides, we already have the same changelog [here]( https://github.com/microsoft/appcenter-sdk-apple/blob/develop/CHANGELOG.md#version-401-under-development). Let's keep it as is.